observability/logging/logging_release_notes/cluster-logging-deploying.adoc

```diff
@@ -0,0 +1,48 @@
+:_mod-docs-content-type: ASSEMBLY
```

Comment:
This assembly does not belong in the 6.0, 6.1, or the 6.2 docs? This is intended?